packages/cadl-rlc-emitter/test/integration/generated/models/propertyTypes/package.json

```diff
@@ -1,20 +1,20 @@
 {
-  "name": "@msinternal/enum-properties",
+  "name": "@msinternal/property-types",
   "sdk-type": "client",
```

Comment:
As this [pr ](https://github.com/Azure/cadl-ranch/pull/86)commented the removed cases will be replaced with in new one `property-types`

Reply:
ok